src/vm/jitinterface.cpp

```diff
@@ -7033,9 +7033,7 @@ bool getILIntrinsicImplementationForUnsafe(MethodDesc * ftn,
         methInfo->options = (CorInfoOptions)0;
         return true;
     }
-    else if ((tk == MscorlibBinder::GetMethod(METHOD__UNSAFE__AS_REF)->GetMemberDef()) ||
```

Comment:
Was it necessary to remove this condition? Will it never be true because of the changes here?